features/addons-esh/src/main/feature/feature.xml

```diff
@@ -47,6 +47,11 @@
         <feature>esh-binding-lirc</feature>
     </feature>
 
+    <feature name="openhab-binding-mqtt" description="MQTT Binding" version="${project.version}">
```

Comment:
Do we need dependency on `<feature>openhab-transport-mqtt</feature>`?

Reply:
Definitely not. Esh mqtt transport is the new revamped one with asynchronous API and the openhab mqtt transport is the one from openHAB 1.x days.